libraries/chain/block_header_state.cpp

```diff
@@ -23,6 +23,54 @@ namespace eosio { namespace chain {
       }
    }
 
+   block_header_state_core::block_header_state_core( uint32_t last_final_blk_ht,
+                                      std::optional<uint32_t> final_on_strong_qc_blk_ht,
+                                      std::optional<uint32_t> last_qc_blk_ht )
```

Comment:
maybe spell out `block_height` everywhere (instead of `blk_ht`)so it is consistent throughout (like on line 34).